src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeThankYouRiskCellConfigurator.swift

```diff
@@ -79,4 +77,11 @@ final class HomeThankYouRiskCellConfigurator: HomeRiskCellConfigurator {
 		cell.titleLabel.accessibilityTraits = .header
 	}
 
+	// MARK: Hashable
+
+	func hash(into hasher: inout Swift.Hasher) { }
+
+	static func == (lhs: HomeThankYouRiskCellConfigurator, rhs: HomeThankYouRiskCellConfigurator) -> Bool {
+		true
```

Comment:
Same as above

Reply:
see my answer for `HomeFindingPositiveRiskCellConfigurator`